File: demos/nodejs/laymine.js
const ms = require('ms-toollib');

let board = ms.laymine(8, 8, 10, 0, 0);
console.log(board);

let board_op = ms.laymine_op(8, 8, 10, 0, 0);
console.log(board_op);

let board_solvable = ms.laymine_solvable(8, 8, 10, 0, 0);
console.log(board_solvable);

File: demos/python/laymine.py
import ms_toollib as ms

help(ms.laymine)
board = ms.laymine(row=16, column=30, mine_num=99, x0=0, y0=0)
print(board)

help(ms.laymine_op)
board_op = ms.laymine_op(row=16, column=30, mine_num=99, x0=0, y0=0)
print(board_op)
